platform.md

```diff
@@ -445,6 +449,47 @@ Usage:
 ##### Layer Restoration
 lifeycle MUST use the provided `cache-dir` or `cache-image` to retrieve cache contents. The [rules](https://github.com/buildpacks/spec/blob/main/buildpack.md#layer-types) for restoration MUST be followed when determining how and when to store cache layers.
 
+#### `image-extender`
+
+The platform MAY execute `image-extender`.
+
+Usage:
+```
+/cnb/lifecycle/image-extender \
+  [-group <group>] \
+  [-exts <exts>] \
+  [-output <output>] \
+  [-log-level <log-level>] \
+  [-plan <plan>]
+```
+
+##### Inputs
```

Comment:
I think we may need `/workspace | CNB_APP_DIR` as well, since they are allowed/expected to read from that directory.